### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by 1GORDON**
# In this project, I achieved the following

- Initialised React app.
- Added React-Redux (npm install react-redux).
- Structured your application files using a "feature folder" approach and use the ducks pattern for your Redux files.
- Added React Router and set two Routes and Links for the app's navigation:
- Books should display the list of books with the Remove button, should have a form for adding a book
- Categories should display "Under construction" text only.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          <div className="nav-links">
+            <Link to="/">Book</Link>
+            <div />
+            <Link to="/categories">Categories</Link>
+          </div>
+        </nav>
+      </header>
+      <Routes>
+        <Route path="/" element={<BooksPage bookList={demoList} />} />
+        <Route path="/categories" element={<CategoriesPage />} />
```
Comment (Emmyn5600) [inline src/App.js]: - [x] Let us restructure the code.

- I would recommend creating a new file called `navbar` that will keep our nav elements.
- Let us the app.js keep the routes only
- The sample data that have been added in the app.js, let us add them inside the component directory (book.js file)

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export default Book;
+
+Book.propTypes = {
+  book: PropTypes.shape({
+    id: PropTypes.number,
+    title: PropTypes.string,
+    author: PropTypes.string,
+  }).isRequired,
+};
```
Comment (Emmyn5600) [inline src/components/Book.js]: - [x] Let us fix the following warnings

testing-2

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @1GORDON 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.


 ## Required Changes ♻️
 
- _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Godanah, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**